src/pages/my_page/user_profile/UserOrcid.tsx

```diff
@@ -18,11 +18,44 @@ import { UrlPathTemplate } from '../../../utils/urlPaths';
 import { authenticatedApiRequest } from '../../../api/apiRequest';
 import { getValueByKey } from '../../../utils/user-helpers';
 import { fetchPerson } from '../../../api/cristinApi';
+import { postOrcidCredentials } from '../../../api/orcidApi';
+import { OrcidCredentials } from '../../../types/orcid.types';
 
 interface UserOrcidProps {
   user: User;
 }
 
+function allPropertiesAreInCredentialsObject(rawCredentials: { [p: string]: string }) {
+  return (
+    rawCredentials.hasOwnProperty('expires_in') &&
+    isNaN(+rawCredentials['expires_in']) &&
```

Comment:
skulle ikke denne vært motsatt? Hvis det er NaN så betyr det at verdien mangler, og funksjonen skulle returnert `false`? Samme for `tokenId` under her
```suggestion
    !isNaN(+rawCredentials['expires_in']) &&
```